rules/android_binary_internal/r8.bzl

```diff
@@ -104,6 +106,7 @@ def process_r8(ctx, jvm_ctx, packaged_resources_ctx, build_info_ctx, **_unused_c
         name = "r8_ctx",
         value = struct(
             final_classes_dex_zip = dexes_zip,
+            proguard_mappings_file = proguard_mappings_file,
```

Comment:
This will need to be bundled up into the AAB later either directly or by reading this file from `ProguardMappingInfo`.

Reply:
there's also the matter of hooking this into native android_binary's proguard map implicit output, which was just recently done for the "discrete" style dex/desugar/optimize pipeline: https://github.com/bazelbuild/bazel/commit/3faa6861b5f2ed8c725815f27f9814fb0d3d6734